docs/ingestion/input-sources.md

```diff
@@ -887,7 +891,7 @@ Compared to the other native batch input sources, SQL input source behaves diffe
 
 The Combining input source lets you read data from multiple input sources.
 It identifies the splits from delegate input sources and uses a worker task to process each split.
-Use the Combining input source only if all the delegates are splittable and can be used by the [Parallel task](./native-batch.md).
+Use the Combining input source only if all the delegates are splittable and can be used by the [parallel task](./native-batch.md).
```

Comment:
this reads awkwardly 😬  I know you were just fixing case.

Reply:
```suggestion
Each delegate input source must be splittable and compatible with the [parallel task type](./native-batch.md).
```